impexp-config/src/main/java/org/citydb/config/project/kmlExporter/KmlExporter.java

```diff
@@ -178,8 +180,10 @@
 	private boolean createGltfModel;
 	private String pathOfGltfConverter;
 	private boolean notCreateColladaFiles;
-	private boolean embedTexturesInGltfFiles;
 	private boolean exportAsKmz;
+	private boolean embedTexturesInGltfFiles;
+	private boolean exportGltfBinary;
+	private boolean enableGltfDracoCompression;
```

Comment:
Moved to branch [glb-draco-support](https://github.com/3dcitydb/importer-exporter/tree/glb-draco-support).